<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MaskBrushSelectionWidget</class>
 <widget class="QWidget" name="MaskBrushSelectionWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>180</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="descriptionLabel">
     <property name="text">
      <string>Brush mode: Use the mouse to modify the mask.
Left click to add to the mask, right click to erase from it.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="brushSizeLabel">
       <property name="text">
        <string>Brush Size:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="brushSizeSlider">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>50</number>
       </property>
       <property name="value">
        <number>15</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="tickPosition">
        <enum>QSlider::TicksBelow</enum>
       </property>
       <property name="tickInterval">
        <number>5</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="brushSizeSpinBox">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>50</number>
       </property>
       <property name="value">
        <number>15</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="showCircleCheckbox">
     <property name="text">
      <string>Show brush circle while hovering</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="allowEmptyMaskCheckbox">
     <property name="text">
      <string>Allow empty masks (preserve mask when all pixels are erased)</string>
     </property>
     <property name="toolTip">
      <string>When enabled, masks will be preserved even when all pixels are erased, useful for creating negative labels for semantic segmentation</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="infoLabel">
     <property name="text">
      <string>Note: The same brush size applies to both adding and erasing operations.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="styleSheet">
      <string>font-style: italic; color: #666;</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui> 